nexus/social_media/services.py

```diff
@@ -0,0 +1,166 @@
+# Third Party Stuff
+import requests
```

Comment:
Did you run `pre-commit` hooks before making the commit? Should not `requests` be present in `Standard` imports?

Reply:
Yeah I forgot to run `pre-commit`. Is there any workout that can be done now?

Reply:
Fix it with another commit.